src/uikit.cs

```diff
@@ -3437,12 +3437,26 @@ interface UIApplicationDelegate {
 		[Export ("application:viewControllerWithRestorationIdentifierPath:coder:")]
 		UIViewController GetViewController (UIApplication application, string [] restorationIdentifierComponents, NSCoder coder);
 
+		[Deprecated (PlatformName.iOS, 13, 2, message: "Use 'ShouldSaveSecureApplicationState' instead")]
+		[Deprecated (PlatformName.TvOS, 13, 2, message: "Use 'ShouldSaveSecureApplicationState' instead")]
```

Comment:
messages should end with `.`
I thought that the introspection tests check the format... c.c. @VincentDondain